benchmarks/Lua/towers.lua

```diff
@@ -0,0 +1,83 @@
+-- This code is derived from the SOM benchmarks, see AUTHORS.md file.
+--
+-- Copyright (c) 2016 Francois Perrad <francois.perrad@gadz.org>
+--
+-- Permission is hereby granted, free of charge, to any person obtaining a copy
+-- of this software and associated documentation files (the 'Software'), to deal
+-- in the Software without restriction, including without limitation the rights
+-- to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+-- copies of the Software, and to permit persons to whom the Software is
+-- furnished to do so, subject to the following conditions:
+--
+-- The above copyright notice and this permission notice shall be included in
+-- all copies or substantial portions of the Software.
+--
+-- THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+-- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+-- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+-- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+-- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+-- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+-- THE SOFTWARE.
+
+local towers = {} do
+setmetatable(towers, {__index = require'benchmark'})
+
+local function create_disk (size)
+    return {size = size, next = nil}
+end
+
+function towers:benchmark ()
```

Comment:
I noticed that the whitespace is inconsistent from time to time.

Does a space belong here after `benchmark` or not?
There isn't one in a few other places, for instance on line 51 for `pop_disk_from`: https://github.com/smarr/are-we-fast-yet/pull/30/commits/fc2922c313a5a5c1f6f7ef6286093235dc1e70fd#diff-cb8e2d36e399aef9ea828138a6378a00R51